process/elasticproc/logsevents/delegatorsProcessor.go

```diff
@@ -75,13 +75,15 @@ func (dp *delegatorsProc) processEvent(args *argsProcessEvent) argOutputProcessE
 	// topics[4] = true - if the delegator was deleted in case of withdrawal OR the contract address in case of delegate operations from staking v3.5 (makeNewContractFromValidatorData, mergeValidatorToDelegationSameOwner or mergeValidatorToDelegationWithWhitelist)
 	activeStake := big.NewInt(0).SetBytes(topics[1])
 
-	contractAddr := dp.pubkeyConverter.Encode(args.logAddress)
+	contractAddr, _ := dp.pubkeyConverter.Encode(args.logAddress)
```

Comment:
To don't have all these pairs **`encodedAddress, _`** you can add a wrapper function in the package 
`ElrondNetwork/elastic-indexer-go/process/elasticproc/converters` that has 2 parameters, one for the public key converter and the second one for the address, and in case of a problem it logs the error.